cmd/stdiscosrv/etc/linux-systemd/discosrv.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Syncthing Discovery Server
+After=network.target
+Documentation=man:stdiscosrv(1)
+
+[Service]
+WorkingDirectory=/var/lib/syncthing-stdiscosrv
+EnvironmentFile=/etc/default/syncthing-stdiscosrv
+ExecStart=/usr/bin/strelaysrv $DISCOSRV_OPTS
+
+# Hardening
+User=syncthing-stdiscosrv
+Group=syncthing
```

Comment:
Seems like a pretty arbitrary choice and as far as I understand the user needs to know it and create those user, that's not automatically taken care of, is it?

Reply:
Normally the debian file takes care of it, but I'll have to look up if this is possible with fpm, running as root is definitely not a good idea

Reply:
That'd be nice, but probably overkill. I guess it's fine if the service fails due to missing user. But then maybe omit the Group entry or make it identical to the user entry so they don't need to setup both a user and a group?

Reply:
I don't know which is best, but another possibility to create new users with Systemd is to drop a file in `/usr/lib/sysusers.d/`.
See for example how the Arch Linux package does it: https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/syncthing

Reply:
If you have to create a user that would be quite annoying, so I'll look up if that works

Reply:
Well I did a good old `preinst` file that does this simply with `adduser` and `addgroup`

Reply:
I tried it out, works like a charm